hack/bundle-gen.py

```diff
@@ -129,26 +130,59 @@ def get_params():
 
     return args
 
+# Traverse through version directories to detect highest version
+def get_previous_version(work_dir, channel_name):
+    upstream_repo = "git@github.com:redhat-openshift-ecosystem/community-operators-prod.git"
```

Comment:
We can const this (and get rid of [COMMUNITY_OPERATORS_HIVE_PKG_URL](https://github.com/openshift/hive/pull/2056/files#diff-625239517ac654a0a60ffdcc20e3a62f806a20df37b553a2ce9e700b57db8176R29), which this logic replaces)